web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -34,11 +36,15 @@
   module.controller('GnCatController', [
     '$scope', '$http', '$q', '$rootScope', '$translate',
     'gnSearchManagerService', 'gnConfigService', 'gnConfig',
-    'gnGlobalSettings', '$location', 'gnUtilityService', 'gnSessionService',
+    'gnGlobalSettings', '$location', 'gnUtilityService', 'gnSessionService', 'gnAdminMenu',
     function($scope, $http, $q, $rootScope, $translate,
             gnSearchManagerService, gnConfigService, gnConfig,
-            gnGlobalSettings, $location, gnUtilityService, gnSessionService) {
+            gnGlobalSettings, $location, gnUtilityService, gnSessionService,gnAdminMenu) {
       $scope.version = '0.0.1';
+      //Display or not the admin menu
+      if ($location.absUrl().indexOf('admin')!=-1){
```

Comment:
absUrl may contain "admin" in pages like catalog.search and catalog.edit. To display the menu only for in the admin.console page, maybe $location.path() === '/admin.console' would be better ?